rotkehlchen/chain/ethereum/modules/eth2/structures.py

```diff
@@ -237,7 +239,7 @@ def process(
             timestamp=self.timestamp,
             asset=A_ETH2,
             amount=amount,
-            taxable=True,
+            taxable=not eth_staking_taxable_after_withdrawal_enabled,
```

Comment:
Please read what I wrote in the previous review carefully: https://github.com/rotki/rotki/pull/4503#discussion_r907568116

> you should just omit emitting the acquisition event all together. Since the goal here is to count the ETH2 as acquired and taxable from the moment the merge happens.

You already went that way with the other place in the code. Why not do the same here? This will result in creating the ETH2 acquisition events and not making them taxable which in the end of the day is wrong.